src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -1365,6 +1369,43 @@ def start_modeling_worksheet(request, format=None):
     ], area_of_interest, user)
 
 
+@swagger_auto_schema(method='post',
+                     request_body=schemas.MODELING_REQUEST,
+                     responses={200: schemas.JOB_STARTED_RESPONSE})
+@decorators.api_view(['POST'])
+@decorators.authentication_classes((SessionAuthentication,
+                                    TokenAuthentication, ))
+@decorators.permission_classes((IsAuthenticated, ))
+@decorators.throttle_classes([BurstRateThrottle, SustainedRateThrottle])
+@log_request
+def start_modeling_gwlfe_prepare(request, format=None):
+    """
+    Starts a job to prepare an input payload for GWLF-E for a given area.
+
+    Given an area of interest or a WKAoI id, gathers data from land, soil type,
+    groundwater nitrogen, available water capacity, K-factor, slope, soil
+    nitrogen, soil phosphorus, base-flow index, and stream datasets.
+
+    By default, NLCD 2019 and NHD High Resolution Streams are used to prepare
+    the input payload. This can be changed using the `layer_overrides` option.
+
+    Only one of `area_of_interest` or `wkaoi` should be provided. If both are
+    given, the `area_of_interest` will be used.
+
+    The `result` should be used with the gwlf-e endpoint.
```

Comment:
This is the one line in the docs that is slightly confusing to me. It's unclear what "the gwlf-e endpoint" is in this context. is that this endpoint (`/modeling/gwlf-e/prepare/`)? or are you referring to a different one?

I know that the "job" field in this response is used to get the results from the jobs endpoint, is that what you're referring to? If so, I think that could be a little more clear. Feel free to set me straight if I'm missing something.

Reply:
Ah good catch, this was a placeholder item, but as of #3483 we know it is the `/gwlf-e/run` endpoint. I'll change the wording.